scripts/index.js

```diff
@@ -0,0 +1,195 @@
+// TEMP: INCREMENT, DECREMENT
+
+const state = {
+    tempCount: 0,
+  };
+
+const increaseTemp = (event) => {
+    state.tempCount += 1
+    // const count = document.querySelector("#tempCount");
+    // count.textContent = `Temperature: ${state.tempCount}`;
+    const count = document.querySelector("#num");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount <= 49) {
+        colorBlue();
+        freezingDay();
+    } else if ((50 <= state.tempCount) && (state.tempCount <= 59)) {
+        colorGreen();
+        coldDay();
+    } else if ((60 <= state.tempCount) && (state.tempCount <= 69))  {
+        colorYellow();
+        coolDay();
+    } else if ((70 <= state.tempCount) && (state.tempCount <= 79)) {
+        colorOrange();
+        warmDay();
+    } else if (state.tempCount >= 80) {
+        colorRed();
+        hotDay();
+    }
+  };
+  
+const decreaseTemp = (event) => {
+    state.tempCount -= 1;
+    const count = document.querySelector("#num");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount <= 49) {
+        colorBlue();
+        freezingDay();
+    } else if ((50 <= state.tempCount) && (state.tempCount <= 59)) {
+        colorGreen();
+        coldDay();
+    } else if ((60 <= state.tempCount) && (state.tempCount <= 69))  {
+        colorYellow();
+        coolDay();
+    } else if ((70 <= state.tempCount) && (state.tempCount <= 79)) {
+        colorOrange();
+        warmDay();
+    } else if (state.tempCount >= 80) {
+        colorRed();
+        hotDay();
+    }
+  };
+
+// TEMP COLORS
+
+const colorBlue = (event) => {
+    const currentTemp = document.querySelector("#num"); // HTML element event will occur on
+    currentTemp.className = `blue`;
+  };
+
+const colorGreen = (event) => {
+    const currentTemp = document.querySelector("#num"); 
+    // currentTemp.className = `${currentTemp.className} green`; // don't carry classes and their data over
+    currentTemp.className = `green`;
+  };  
+
+const colorYellow = (event) => {
+    const currentTemp = document.querySelector("#num");
+    currentTemp.className = `yellow`;
+  };  
+
+const colorOrange = (event) => {
+    const currentTemp = document.querySelector("#num");
+    currentTemp.className = `orange`;
+  }; 
+
+const colorRed = (event) => {
+    const currentTemp = document.querySelector("#num");
+    currentTemp.className = `red`;
+  };
```

Comment:
Not that each of the *color* functions does the work of finding the element with the id `#num`. Consider refactoring so that this element is passed in as a parameter and only needs to be found once.